apps/staking/app/layout.tsx

```diff
@@ -33,36 +22,28 @@ const buildInfo = getBuildInfo();
 
 export default async function RootLayout({ children }: { children: ReactNode }) {
   const { locale, direction, messages } = await getLocalizationData();
-  const initialWagmiState = cookieToInitialState(wagmiConfig, (await headers()).get('cookie'));
+  const wagmiCookie = (await headers()).get('cookie');
 
   return (
     <html
       lang={locale}
       dir={direction}
       className={cn(RobotoFlex.variable, MonumentExtended.variable)}
     >
-      <FeatureFlagProvider>
-        <GlobalProvider
-          messages={messages}
-          locale={locale}
-          initialState={initialWagmiState}
-          wagmiMetadata={wagmiMetadata}
-        >
-          <body className="bg-session-black font-roboto-flex text-session-text overflow-x-hidden">
-            <ChainBanner />
-            <Suspense>
-              <RemoteBanner />
-            </Suspense>
-            <Suspense>
-              <Header />
-            </Suspense>
-            <main>{children}</main>
-            {!isProduction ? <DevSheet buildInfo={buildInfo} /> : null}
-            <TOSHandler />
-          </body>
+      <GlobalProvider messages={messages} locale={locale} wagmiCookie={wagmiCookie}>
+        <body className="bg-session-black font-roboto-flex text-session-text overflow-x-hidden">
+          {/*<ChainBanner />*/}
```

Comment:
What's the plan here?

Reply:
Left there for the future, rather than deleting. as we might want to reuse the code